README.rst

```diff
@@ -87,3 +87,19 @@ Follow the same instructions as above except for the following:
 3. **Copy the OpenSSL folder**
 
 4. **Build the amalgamation and install with the latest Python x64**
+
+SQLCipher compatibility issues
+----------------------------
+The encryption has default compatibility with SQLCipher version, installed on your machine (current SQLCipher version 4).
```

Comment:
Doc is generally good but the grammar and wording needs cleaned up. 
1) "has default compatibility with the SQLCipher version installed on your machine"
2) Drop the comment in parenthesis about the current version because it dates this doc unnecessarily

Reply:
free english coaching. Thank you